jaseci_kit/jaseci_kit/modules/entity_extraction_type_2/README.md

```diff
@@ -344,5 +353,131 @@
+                precision    recall  f1-score   support
+
+            O       0.00      0.00      0.00         0
+        I-PER       0.91      0.90      0.91      2496
+        I-ORG       0.82      0.68      0.74      1018
+        I-MISC      0.84      0.54      0.66       236
+        I-LOC       0.66      0.64      0.65       252
+        B-PER       0.84      0.85      0.85      2714
+        B-ORG       0.83      0.71      0.77      2513
+        B-MISC      0.82      0.64      0.72       991
+        B-LOC       0.85      0.84      0.85      1965
+
+     accuracy                           0.79     12185
+    macro avg       0.73      0.65      0.68     12185
+    weighted avg    0.85      0.79      0.82     12185
+
+    2022-06-01 07:08:27.092694     ------------------------------------------------------------
+    2022-06-01 07:08:27.092707     Total time taken to completed testing :  0:00:04.294899
+    2022-06-01 07:08:27.092722     ------------------------------------------------------------ 
+
+    ```
+## **4. Predicting Entities**
+* For predicting entities we are going execute walker `predict_entity_from_tfm` and provide text input in context and will get output `list of entities` available in text-data.
+Run the following command to execute walker `predict_entity_from_tfm`
+
+    ```
+    walker run predict_entity_from_tfm -ctx "{\"text\":\"It was the second costly blunder by Syria in four minute\"}"
+    ```
+* After executing walker `predict_entity_from_tfm` , will get `predicted entities` list as
+    ```
+    [
+        {
+            "entity_value": "syria",
+            "entity_type": "B-LOC",
+            "score": 0.8613966703414917,
+            "start_index": 36,
+            "end_index": 41
+        }
+    ]
+    ```
+
+# **5. Experiments and methodology**
+Let us further look into our model training and evaluation methodology
+
+## **Evaluation of tfm_ner with `tiny-bert` and `bert`**
+We are evaluating `tiny-bert` and `bert-base-uncased` model on following datasets.
+
+
+## **Dataset description**
+We are using two different dataset.
+
+**1. CONLL2003 [dataset](https://huggingface.co/datasets/conll2003)** is a named entity recognition dataset released as a part of CoNLL-2003 shared task. CoNLL-2003 dataset have **4 labels**(PER, ORG, LOC, MISC).
+
+**2. FEW-NERD [dataset](https://ningding97.github.io/fewnerd/)** is a large-scale, fine-grained manually annotated named entity recognition dataset, which contains `8 coarse-grained(Major)` types, `66 fine-grained(All)` types labels.
+
+### **Dataset details**
+| Dataset Name                   | train dataset             | validation dataset         | test dataset          |
+|--------------------------------|---------------------------|----------------------------|-----------------------|
+| Conll2003	                     | 14041                     | 3250	                      | 3453                  |
```

Comment:
Make these numbers scientific representation, e.g. 14,041 with the comma.

Reply:
changes done